internal/consumer/http.go

```diff
@@ -0,0 +1,61 @@
+package consumer
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+type DataService interface {
+	getTopStories() ([]int, error)
+	getItem(id int) (*Item, error)
+}
+
+type HttpService struct {}
```

Comment:
This mix names between DataService and HttpService I think is a bit confusing

Reply:
yep i agree. I have refactored multiple interface names to better accommodate go best practices